libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFKeyStoreKey.m

```diff
@@ -54,11 +60,47 @@ - (void)encodeWithCoder:(NSCoder *)aCoder
     [aCoder encodeObject:self.encryptionKey forKey:kKeyStoreKeyDataArchiveKey];
 }
 
-- (id)copyWithZone:(NSZone *)zone
+- (instancetype)copyWithZone:(NSZone *)zone
 {
     SFKeyStoreKey *keyCopy = [[[self class] allocWithZone:zone] init];
     keyCopy.encryptionKey = [self.encryptionKey copy];
     return keyCopy;
 }
 
++ (nullable instancetype)fromKeyChain:(NSString*)keychainId archiverKey:(NSString*)archiverKey
+{
+    SFKeyStoreKey* keyStoreKey;
+    SFKeychainItemWrapper *keychainItem = [SFKeychainItemWrapper itemWithIdentifier:keychainId account:nil];
+    NSData *keyStoreKeyData = [keychainItem valueData];
+    if (keyStoreKeyData == nil) {
+        return nil;
+    } else {
+        NSKeyedUnarchiver *unarchiver = [[NSKeyedUnarchiver alloc] initForReadingWithData:keyStoreKeyData];
+        keyStoreKey = [unarchiver decodeObjectForKey:archiverKey];
+        [unarchiver finishDecoding];
+        
+        return keyStoreKey;
+    }
+}
+
+- (OSStatus) toKeyChain:(NSString*)keychainId archiverKey:(NSString*)archiverKey
+{
+    SFKeychainItemWrapper *keychainItem = [SFKeychainItemWrapper itemWithIdentifier:keychainId account:nil];
```

Comment:
That code was previously (duplicated) in SFGeneratedKeyStore and SFPasscodeKeyStore